### PR TITLE
#2056: Show Examiner if Resume has one

### PR DESCRIPTION
**Pull request by sbzDev**
https://github.com/zerocracy/farm/issues/2056

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#2057](https://github.com/zerocracy/farm/issues/2057) is now in scope, role is `REV`

**Comment by sbzDev** [comment]
@g4s8 is PR ok?

**Comment by sbzDev** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/2057#issuecomment-497080563))

@sbzDev This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#2057](https://github.com/zerocracy/farm/issues/2057) is [in scope](http://www.zerocracy.com/policy.html#14) for 7hr
  * The role is `REV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @sbzdev[/z](https://www.0crat.com/u/sbzdev): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%232057%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

File: src/main/resources/xsl/resume.xsl [hunk 1]
```diff
@@ -56,6 +56,14 @@ SOFTWARE.
               <xsl:value-of select="personality"/>
             </code>
           </li>
+          <xsl:if test="examiner">
```
Comment (g4s8) [inline src/main/resources/xsl/resume.xsl]: @sbzDev can we show something like "Your resume will be assigned for review soon" if examiner is empty?

File: src/main/java/com/zerocracy/tk/TkJoin.java [hunk 2]
```diff
@@ -59,19 +63,32 @@ public Response act(final RqRegex req) throws IOException {
...
+                        resumes.examiner(resume.login())
+                    )
+                );
+            }
+            result = new RsPage(
+                this.farm,
+                "/xsl/resume.xsl",
+                req,
+                () -> new XeAppend(
+                "resume",
```
Comment (g4s8) [inline src/main/java/com/zerocracy/tk/TkJoin.java]: @sbzDev looks like wrong indentation here, can we use `() -> new XeAppend("resume", items)` instead?

File: src/main/resources/xsl/resume.xsl (hunk 1 above)
Comment (sbzDev) [reply src/main/resources/xsl/resume.xsl]: Good idea! applying fix

File: src/main/java/com/zerocracy/tk/TkJoin.java (hunk 2 above)
Comment (sbzDev) [reply src/main/java/com/zerocracy/tk/TkJoin.java]: tried it and got "Checkstyle: src/main/java/com/zerocracy/tk/TkJoin.java[87]: Parameters should start on a new line (BracketsStructureCheck)"

**Comment by g4s8** [comment]
@sbzDev thanks

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2057` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#2057](https://github.com/zerocracy/farm/issues/2057) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by sbzDev** [comment]
@g4s8 what does it mean "Job gh:zerocracy/farm#2057 is not assigned, can't get performer" ???

**Comment by g4s8** [comment]
@sbzDev it means that this job (PR #2057) was not assigned for review

**Pull request merged**